base_business_document_import/tests/test_business_document_import.py

```diff
@@ -164,11 +164,6 @@ def test_match_product(self):
         )
         self.assertEqual(res, product1)
         raise_test = True
-        try:
```

Comment:
This change makes little sense to me. You are not testing anything anymore.
Moreover, for testing exceptions we should use `assertRaises` and not catch them.

You should test that you have the right msg instead.

Reply:
Thanks for your help, writing test is still a bit complicated for me :)
Let's hope this will do the trick.

Reply:
@simahawk Can you re-review?